autonomy/chain/base.py

```diff
@@ -20,10 +20,83 @@
 """Chain interaction base classes."""
 
 from enum import Enum
+from typing import Optional
+
+from aea.configurations.data_types import PublicId
+from aea.contracts.base import Contract
+
+from autonomy.chain.constants import (
+    AGENT_REGISTRY_CONTRACT,
+    COMPONENT_REGISTRY_CONTRACT,
+    CONTRACTS_DIR_FRAMEWORK,
+    CONTRACTS_DIR_LOCAL,
+    REGISTRIES_MANAGER_CONTRACT,
+)
 
 
 class UnitType(Enum):
     """Unit type."""
 
     COMPONENT = 0
     AGENT = 1
+
+
+class RegistryContracts:
```

Comment:
This is not a really good idea will have to come up with a better idea for a collection of contracts so we don't have to repeat logic to load contracts

Reply:
Why not a dictionary object which can be imported and is populated once at loading time?